instrumentation/opentelemetry-api-metrics-1.0/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/opentelemetryapi/metrics/MeterTest.java

```diff
@@ -333,8 +334,10 @@ void longValueRecorder() {
                       .hasDescription("d")
                       .hasUnit("u")
                       .hasInstrumentationLibrary(
-                          InstrumentationLibraryInfo.create(instrumentationName, "1.2.3"));
-                  assertThat(metric.getDoubleSummaryData().getPoints())
+                          InstrumentationLibraryInfo.create(instrumentationName, "1.2.3"))
+                      // TODO(anuraaga): Switch to histogram in future version?
```

Comment:
Actually I think I need to fix the reverse OTLP parse to use histogram, not summary. Will check it

Reply:
I don't think we have histograms as the default aggregation in 1.5.0. I think that comes in 1.6.0: https://github.com/open-telemetry/opentelemetry-java/pull/3491

Reply:
Ah - let me know if I should revert the change to OpenTelemetryInstaller that I have in this PR.

Reply:
yeah, we should probably keep that part in place until 1.6.0 hits, rather than thrash back to summary, then back to histogram again.